src/components/CompanyHomePage/MatchCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Button, Image, Badge } from 'react-bootstrap'
+import male from '../../media/male.png'
+import SearchBar from './SearchBar'
+const MatchCard = () => {
+  return (
+    <>
+      <SearchBar />
```

Comment:
Put the MatchCard inside the components folder. 
The search bar is still inside the card and therefore will be above every jobseeker match info.